pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/ActivityInstanceDao.java

```diff
@@ -240,6 +255,18 @@ default void saveSubstitutions(long instanceId, Map<String, String> substitution
         DBUtils.checkInsert(variables.size(), inserted.length);
     }
 
+    @SqlQuery("select distinct sub.variable_name"
+            + "from activity_instance_substitution as sub"
+            + "join activity_instance as ai on ai.activity_instance_id = sub.activity_instance_id"
+            + "join (select rev.start_date, rev.end_date"
+            + "        from activity_version as ver"
```

Comment:
Might be missing spaces in 3 lines in split lines here.

Reply:
Good catch! Thanks!

Reply:
Fixed